app/assets/javascripts/Components/assessment_chart.jsx

```diff
@@ -0,0 +1,215 @@
+import React from "react";
+import {Bar} from "react-chartjs-2";
+import PropTypes from "prop-types";
+import {chartScales} from "./Helpers/chart_helpers";
+import ReactTable from "react-table";
+
+class AssessmentChart extends React.Component {
```

Comment:
With the way I redesigned this, I wonder if it might be better to split the assessment graph and grade breakdown graph into two separate helper components.